### ISSUE TITLE
[terra-button-group] [terra-toolbar] Selected button within button-group blends into terra-toolbar background color in lowlight theme

### ISSUE THREAD

**Issue by juliacalandro** [cerner/terra-core#3413]
# Bug Report

## Description
In our component, we have the need to display a terra-button-group component within a terra-toolbar. When the clinical lowlight theme is applied and one of the buttons within the button-group is selected, the styling of that button blends into the background of the terra-toolbar component, so it almost looks as if the button is disabled.

Since it seems that one of the primary use-cases of the toolbar is to display buttons, ideally this styling would be optimized automatically by Terra so that the consumer does not have to apply custom lowlight styling.

## Steps to Reproduce
1. Implement a React component that has a terra-button-group within a terra-toolbar.
2. Apply the clinical lowlight theme to the component.
3. Select one of the buttons of the button-group to trigger the selected styling.

## Additional Context / Screenshots
![Screen Shot 2021-03-29 at 1 08 04 PM](https://user-images.githubusercontent.com/39311706/112891083-4feb0680-909d-11eb-9e51-938278804c75.jpg)

## Expected Behavior
The selected button should be clearly visible and distinguishable from the rest of the toolbar in the lowlight theme.

## Environment
* Component Name and Version: terra-toolbar - ^1.20.0, terra-button-group - ^3.26.0
* Browser Name and Version: Google Chrome v89.0.4389.82 (Official Build) (x86_64)
* Node/npm Version: lts/dubnium
* Webpack Version: ^4.30.0
* Operating System and version (desktop or mobile): macOS Mojave Desktop
